app/_hub/kong-inc/request-transformer/index.md

```diff
@@ -185,16 +220,36 @@ $ curl -X POST http://localhost:8001/services/example-service/plugins \
   --data "config.add.headers=h1:v1"
 ```
 
-incoming request headers | upstream proxied headers:
----           | ---
-h1: v2        | <ul><li>h1: v2</li><li>h2: v1</li></ul>
-h3: v1        | <ul><li>h1: v1</li><li>h2: v1</li><li>h3: v1</li></ul>
+<table>
+  <tr>
+    <th>incoming request headers</th>
+    <th>upstream proxied headers:</th>
+  </tr>
+  <tr>
+    <td>h1: v2</td>
+    <td>
+      <ul>
+        <li>h1: v2</li>
+        <li>h2: v1</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>h3: v1</td>
+    <td>
+      <ul>
+        <li>h1: v1</li>
+        <li>h2: v1</li>
+        <li>h3: v1</li>
+      </ul>
+    </td>
+  </tr>
+</table>
 
-incoming request querystring | upstream proxied querystring
----           | ---
-\?q1=v1       | \?q1=v1&q2=v1
--             | \?q1=v2&q2=v1
```

Comment:
@kikito also, is this row meant to be a new line in the row above?

Reply:
(the row is meant to be empty, as the upstream column shows what happens with an empty request)